pkg/booleanpolicy/default_policies_test.go

```diff
@@ -2759,6 +2760,11 @@ func (suite *DefaultPoliciesTestSuite) TestDropCaps() {
 			storage.BooleanOperator_AND,
 			[]string{"", "MODULE", "PTRACE", "NICE", "MODULE,NICE"},
 		},
+		{
+			[]string{"ALL"},
```

Comment:
If I understood this test correct, this tests the case where "ALL" is added to the policy. What about a test case where policy contains some other values but the deployment drops ALL?

Reply:
Good point. I added a deployment which drops ALL to the test cases. None of the policies should fire a violation on it, is the expectation.